### ISSUE TITLE
Explore GIFs to explaning the upcoming features

### ISSUE THREAD

**Issue by abhi18av** [seqeralabs/nf-tower-docs#56]
@evanfloden I was worndering whether GIFs might help us document and explain the features in a more succint way. 

The current screenshots based methodology would end up increasing the lenth of the page by quite a lot.

**Comment by evanfloden** [comment]
I agree this could be interesting. Can we try with an example and see how it looks?
